sdk/tests/conformance2/extensions/webgl_multisampled_render_to_texture.html

```diff
@@ -0,0 +1,181 @@
+<!--
```

Comment:
Please add any new tests to `00_test_list.txt` in the same directory.

For WebGL 1.0 tests, use `--min-version 1.0.4`.

For WebGL 2.0 tests, use `--min-version 2.0.1`.